File: SDKs/amazonaws.com/fsx/2018-03-01/typescript/src/sdk/models/shared/describefilesystemaliasesresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { Alias } from "./alias";



// DescribeFileSystemAliasesResponse
/** 
 * The response object for <code>DescribeFileSystemAliases</code> operation.
**/
export class DescribeFileSystemAliasesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=Aliases", elemType: Alias })
  aliases?: Alias[];

  @SpeakeasyMetadata({ data: "json, name=NextToken" })
  nextToken?: string;
}

File: SDKs/amazonaws.com/fsx/2018-03-01/python/src/sdk/models/shared/describefilesystemaliasesresponse.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class DescribeFileSystemAliasesResponse:
    r"""DescribeFileSystemAliasesResponse
    The response object for <code>DescribeFileSystemAliases</code> operation.
    """
    
    aliases: Optional[List[Alias]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Aliases') }})
    next_token: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('NextToken') }})
    
